implementation.js

```diff
@@ -0,0 +1,47 @@
+/*! https://mths.be/fromcodepoint v1.0.0 by @mathias */
+
+'use strict';
+
+var ToNumber = require('es-abstract/2019/ToNumber');
+var IsInteger = require('es-abstract/2019/IsInteger');
+var callBound = require('es-abstract/helpers/callBound');
+
+var ArrayPush = callBound('Array.prototype.push');
+var Apply = callBound('Function.prototype.apply');
```

Comment:
typically what i do here is `callBind.apply(GetIntrinsic('%Function.prototype.apply%'), StringFromCharCode)`, which is a bit more verbose but looks more natural at the callsite. this is fine tho.